tests/test_elastic.py

```diff
@@ -397,6 +397,18 @@ def test_unreg_users_dont_show_in_search(self):
         contribs = search.search_contributor(unreg.fullname)
         assert_equal(len(contribs['users']), 0)
 
+
+    def test_unreg_users_do_show_on_projects(self):
```

Comment:
While we're thinking of it, if we don't have the test, could you add one really quick to verify that unregistered contributors don't show up in user search? Obviously this code couldn't possibly do that, but who knows what people may do in the future.